tutor/src/flux/teacher-task-plan.coffee

```diff
@@ -45,6 +45,7 @@ TeacherTaskPlanConfig =
       @emitChange()
 
   addPublishingPlan: (plan, courseId) ->
+    return unless @_local[courseId]?
     addPlan(plan, @_local[courseId])
     @emitChange()
 
```

Comment:
I'll nitpick a teensy bit - how about getting rid of the duplicate lookup, maybe:
```
addPublishingPlan: (publishingPlan, courseId) ->
  existingPlan = @_local[courseId]
  addPlan(publishingPlan, existingPlan) and @emitChange() if existingPlan?
```